.env.sample

```diff
@@ -7,6 +7,7 @@ FEEDBACK_URL=https://example.com/generic-feedback
 FEEDBACK_URL_TABLE=https://example.com/table-feedback
 
 FEATURE_HIDE_DAYS_COLUMNS_IN_INCENTIVES_TABLE=true
+FEATURE_SHOW_ANALYTICS_PC_TRENDS=true
```

Comment:
maybe same for integration test env file?

Reply:
I didn't add it yet as it would default to `false` for the time being and we'll change to `true` when we'll have some integration tests 🤔

Reply:
But I'll add as false now in the `.feature.env` 👌

Reply:
https://github.com/ministryofjustice/hmpps-incentives-ui/pull/190/commits/24f8e8b23465cd8e595ca280dc7aa4bb6e067b5d